lib/src/custom_items/simulation_card.dart

```diff
@@ -1,32 +1,46 @@
 import 'package:flutter/material.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:provider/provider.dart';
+import 'package:simulate/src/simulations/simulations.dart';
 
 class SimulationCard extends StatefulWidget {
   final String simulationName;
   final Widget direct;
   final String image;
   final String infoLink;
+  final int fav;
+  final int id;
   SimulationCard(
       {Key key,
+      @required this.id,
       @required this.simulationName,
       @required this.direct,
       @required this.image,
-      @required this.infoLink})
+      @required this.infoLink,
+      @required this.fav})
       : super(key: key);
 
   _SimulationCardState createState() => _SimulationCardState();
 }
 
 class _SimulationCardState extends State<SimulationCard> {
-  bool fav = false;
+  int fav;
+
   _togglefav() {
     setState(() {
-      fav = !fav;
+      fav *= -1;
```

Comment:
Any specific reason to change this from a `bool` to an `int`?

Reply:
saving to shared prefs...